heartbeat/powervs-subnet.in

```diff
@@ -0,0 +1,1059 @@
+#!@PYTHON@ -tt
+# ------------------------------------------------------------------------
+# Description:  Resource Agent to move a Power Virtual Server subnet
+#               and its IP address from one virtual server instance
+#               to another.
+#
+# Authors:      Edmund Haefele
+#               Walter Orb
+#
+# Copyright (c) 2024 International Business Machines, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------
+
+import ipaddress
+import json
+import os
+import re
+import socket
+import subprocess
+import sys
+import textwrap
+import time
+
+import requests
+import requests.adapters
+import urllib3.util
+
+OCF_FUNCTIONS_DIR = os.environ.get(
+    "OCF_FUNCTIONS_DIR", "%s/lib/heartbeat" % os.environ.get("OCF_ROOT")
+)
+
+sys.path.append(OCF_FUNCTIONS_DIR)
+
+try:
+    import ocf
+except ImportError:
+    print("ImportError: ocf module import failed.", file=sys.stderr)
```

Comment:
CI fails on Debian testing and RHEL8 due to this line.
```
10:16:28    File "/srv/workspace/resource-agents-pipeline_PR-1948/resource-agents/heartbeat/powervs-subnet", line 48
10:16:28      print("ImportError: ocf module import failed.", file=sys.stderr)
10:16:28                                                          ^
10:16:28  SyntaxError: invalid syntax
```

Reply:
O.K. - hopefully  `sys.stderr.write()` is more generic